website/content/docs/enterprise/license/faqs.mdx

```diff
@@ -0,0 +1,102 @@
+
+## Q: What is the impact on EVAL licenses due to this change?
+
+The 6-hour trial period for Enterprise binaries allows these particular binaries to run without a license file and will be deprecated as of Vault 1.8(+). This change means that any clusters deployed with Vault 1.8(+) ENT binaries must have a valid license, and the license must be on the disk (auto-loading).
+
+## Q: Are there any changes to existing methods for manual license loading ( API or CLI)?
+
+There are no significant changes to the API or CLI license loading with Vault 1.8.
+However, the `/sys/license` endpoint is slated for deprecation in a future release.
+Please see the question: [Is there a limited time for support of licenses that are in storage?](#q:-is-there-a-limited-time-for-support-of-licenses-that-are-in-storage?).
+
+## Q: Is there a grace period when Eval licenses expire?
+
+Evaluation licenses have a 1-day grace period. The grace period is the time until the license expires. Upon expiration, Vault will seal and will require a valid license to unseal and function properly.
+
+## Does this affect clients?
+
+No, there is no impact to the Vault agent.
+
+## Q: Are the license files locked to a specific cluster?
+
+The changes to licenses apply to all nodes in a cluster. The license files are not locked to a cluster, but are independently applied to the appropriate clusters.
+
+## Q: Will a EULA check happen every time a Vault restarts?
+
+Yes, starting with Vault 1.8, ENT binaries will be subjected to a EULA check. The release of Vault 1.8 introduces the EULA check for Evaluation licenses (non-Evaluation licenses are evaluated with a EULA check during contractual agreement) .
+Although the agreement to a EULA occurs only once (when the user receives their license), Vault will check for the presence of a valid license every time a node is restarted.
+
+When a customer upgrades existing clusters to a 1.8(+) release, a valid license must exist to upgrade successfully. This valid license must be in storage or on-disk (auto-loaded). Please see the below question: [Is there a limited time for support of licenses that are in storage?](#q:-is-there-a-limited-time-for-support-of-licenses-that-are-in-storage?).
+
+Also, when customers deploy new clusters to a 1.8(+) release, a valid license must exist to upgrade successfully. This valid license must be on-disk (auto-loaded).
+
+## Q: What scenarios should a customer plan for due to these license changes?
+
+- **New Cluster Deployment**: When a customer deploys new clusters to a 1.8(+) release, a valid license must exist to successfully deploy. This valid license must be on-disk (auto-loaded).
+
+- **Eventual Migration**: Since in-storage licenses will not be supported forever, migrating to auto-loaded licenses will be required. Please see the below question: [Is there a limited time for support of licenses that are in storage?](#q:-is-there-a-limited-time-for-support-of-licenses-that-are-in-storage?).
```

Comment:
```suggestion
- **Eventual Migration**: Since in-storage licenses will not be supported forever, migrating to auto-loaded licenses will be required. Please see the below question: [Is there a limited time for support of licenses that are in storage?](#q-is-there-a-limited-time-for-support-of-licenses-that-are-in-storage).
```